.github/workflows/create-changelog.yml

```
@@ -0,0 +1,98 @@
+name: Generate and publish changelog
+
+on:
+  workflow_call:
+    inputs:
+      changelogServiceName:
+        type: string
+        required: true
+      changelogStartVersion:
+        description: "From which version start to building changelog"
+        type: string
+        required: true
```

```suggestion
        required: false
```
And let's a default empty value